src/utils/HashlinkEncoder.ts

```diff
@@ -37,17 +40,16 @@ export class HashlinkEncoder {
    */
   public static encode(
     buffer: Buffer | Uint8Array,
-    hashAlgorithm: 'sha2-256',
-    baseEncoding: BaseName = 'base58btc',
-    metadata?: Metadata
+    metadata?: Metadata,
+    hashAlgorithm: 'sha2-256' = 'sha2-256',
```

Comment:
I'd like to not make `hashAlgorithm` optional. This makes it easier in the future when we support multiple hashing algorithms

Reply:
This doesn't seem resolved @blu3beri